### PR TITLE
Update georesources-report-types.ttl

### PR DESCRIPTION
**Pull request by KellyVance**
Rename of Well Completion Reports, Petroleum Production, and Petroleum REserves and REsources Reports in line with names in regulation
Minor Definition Fixes
addition of internal lodgement collection

### CODE REVIEW

File: vocabularies/georesources-report-types.ttl [hunk 1]
```diff
@@ -362,7 +362,7 @@ grt:petroleum-end-authority a skos:Concept ;
     skos:altLabel "End of Authority Report (DAA)"@en,
         "End of Authority Report (PSL)"@en,
 	"Petroleum Report - End of Authority Report"@en;
-    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority, detailing the work undertaken on the permit throughout the life of tenure."@en  ;
+    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority, detailing the work undertaken on the authority throughout the life of tenure."@en  ;
```
Comment (talebib) [inline vocabularies/georesources-report-types.ttl:365]: Should this also cover Petroleum Survey License (PSL)?

Comment (KellyVance) [reply vocabularies/georesources-report-types.ttl:365]:
```suggestion
    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority or Petroleum Survey License, detailing the work undertaken on the authority throughout the life of tenure."@en  ;

```

Good pick up.

**Review by talebib — APPROVED** [review]
Thanks Vance. Fine by me. Approved.

File: vocabularies/georesources-report-types.ttl [hunk 2]
```diff
@@ -865,3 +866,68 @@ grt:water-reports a skos:Collection ;
...
+        grt:pggd-02,
+        grt:pggd-03,
+        grt:pggd-04,
+        grt:pa-21A,
+        grt:pa-22A,
+        grt:pm-1-2013,
+        grt:pa-42,
+        grt:wra-05A,
+        grt:mmol-44,
+        grt:collaborative-exploration-initiative-proposals
```
Comment (DavidCrosswellGSQ) [inline vocabularies/georesources-report-types.ttl:920]: The concepts from hereon need to have a comma at the end of the line.

**Review by DavidCrosswellGSQ — CHANGES_REQUESTED** [review]
Please rectify errors. Cheers, DC

File: vocabularies/georesources-report-types.ttl [hunk 3]
```diff
@@ -865,3 +867,68 @@ grt:water-reports a skos:Collection ;
...
+        grt:departmental-drilling
+        grt:geological-survey-of-queensland-publication
+        grt:geological-survey-of-queensland-record
+        grt:permit-report-other
+        grt:petroleum-report-other
+        grt:presentation
+        grt:queensland-government-mining-journal
+        grt:technical-report
+        grt:water-report-other
+        grt:any-other-report;
```
Comment (KellyVance) [inline vocabularies/georesources-report-types.ttl:933]:
```suggestion
        grt:collaborative-exploration-initiative-proposals,
        grt:collaborative-exploration-initiative-final,
        grt:commissioned-industry-study-or-report,
        grt:departmental-drilling,
        grt:geological-survey-of-queensland-publication,
        grt:geological-survey-of-queensland-record,
        grt:permit-report-other,
        grt:petroleum-report-other,
        grt:presentation,
        grt:queensland-government-mining-journal,
        grt:technical-report,
        grt:water-report-other,
        grt:any-other-report;   

```
Fixed! Thanks.

**Review by DavidCrosswellGSQ — APPROVED** [review]

**Pull request merged**